cmd/config.go

```diff
@@ -377,7 +380,7 @@ func overWriteConfig(cmd *cobra.Command, cfg *Config) error {
 			}
 
 			// overwrite the config file
-			err = ioutil.WriteFile(viper.ConfigFileUsed(), out, 0666)
+			err = ioutil.WriteFile(viper.ConfigFileUsed(), out, 0600)
```

Comment:
Why `600`?

Reply:
`G306: Poor file permissions used when writing to a new file`